test/cypress/e2e/oidc/oidc_auth_test.spec.js

```diff
@@ -18,22 +18,22 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-const login = 'admin';
-const password = 'admin';
-
 describe('Log in via OIDC', () => {
   afterEach(() => {
-    cy.origin('http://localhost:5601', () => {
-      cy.clearCookies();
-      cy.clearLocalStorage();
-    });
+    cy.clearCookies();
+    cy.clearLocalStorage();
   });
 
   const kcLogin = () => {
-    cy.get('#kc-page-title').should('be.visible');
-    cy.get('input[id=username]').should('be.visible').type(login);
-    cy.get('input[id=password]').should('be.visible').type(password);
-    cy.get('#kc-login').click();
+    cy.origin('http://127.0.0.1:8080', () => {
```

Comment:
qq: Do we need to use `origin` here? Should we visit this and set the origin to `5601`?

Reply:
This needed to be wrapped in `cy.origin('http://127.0.0.1:8080')` since its a different origin then the first request of the test which goes to `localhost:5601`. Without wrapping this, cypress was not searching for the following elements in the correct DOM and was failing to find them.

Reply:
> Without wrapping this, cypress was not searching for the following elements in the correct DOM and was failing to find them.

Got it! Yes, I understand it now. I was thinking the other way around.